documentation/modules/exploit/multi/http/horde_form_file_upload.md

```diff
@@ -0,0 +1,97 @@
+Horde Groupware Webmail is a popular open-source groupware platform written in PHP. A vulnerability allows an authenticated, unprivileged user to create a malicious PHP file under the web root and gain arbitrary code execution on the server. 
+
+## Vulnerable Application
+
+The Horde subcomponent Horde Form < 2.0.19 is affected. This module was specifically tested against Horde 5.2.22 and 5.2.17 with Horde Form < 2.0.19 installed with PEAR on Debian.
+
+## Verification Steps
```

Comment:
These install instructions are great, but should be at an h3 (###) level and titled something similar to "Docker install on Ubuntu xx.xx"